lib/fastlane/plugin/firebase_app_distribution/actions/firebase_app_distribution_action.rb

```diff
@@ -250,11 +219,11 @@ def self.get_upload_token(app_id, binary_path)
         if contact_email.nil? || contact_email.strip.empty?
           UI.crash!(ErrorMessage::GET_APP_NO_CONTACT_EMAIL_ERROR)
         end
-        return CGI.escape("projects/#{response.body[:projectNumber]}/apps/#{response.body[:appId]}/releases/-/binaries/#{binary_hash}")
+        return upload_token_format(response.body[:appId], response.body[:projectNumber], binary_hash)
       end
 
       def self.upload_binary(app_id, binary_path)
-        connection.post("/app-binary-uploads?app_id=#{app_id}", File.open(binary_path).read) do |request|
+        connection.post(binary_upload_path(app_id), File.open(binary_path).read) do |request|
```

Comment:
For the URL helper methods I think "*_url" would be a better naming scheme than "*_path" especially to distinguish from the file path.